astroquery/jplhorizons/core.py

```diff
@@ -1032,17 +1043,21 @@ def vectors_async(self, *, get_query_payload=False,
 
         URL = conf.horizons_server
 
-        # check for required information
+        # check for required information and assemble commandline stub
         if self.id is None:
             raise ValueError("'id' parameter not set. Query aborted.")
+        elif isinstance(self.id, dict):
+            commandline = (
+                f"g:{self.id['lon']},{self.id['lat']},"
+                f"{self.id['elevation']}@{self.id['body']}"
+            )
```

Comment:
this also probably fit the line lenght limit

Reply:
condensed to one line.